browser/src/control/Control.ContextToolbar.ts

```diff
@@ -0,0 +1,173 @@
+/* -*- js-indent-level: 8 -*- */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * L.Control.NotebookbarWriter - definition of notebookbar content in Writer
+ */
+
+/* global _ _UNO app Point */
+
+class ContextToolbar {
+	static showContextToolbar(pos: Point): void {
+		this.hideContextToolbar();
+		const container = L.DomUtil.createWithId(
+			'div',
+			'context-toolbar',
+			document.body,
+		);
+		L.DomUtil.addClass(container, 'notebookbar horizontal');
+		container.style.left = pos.x + 'px';
+		container.style.top = pos.y + 'px';
+		app.map.uiManager.notebookbar.builder.build(
+			container,
+			this.getWriterTextContext(),
+			false,
+		);
+	}
+
+	static hideContextToolbar(): void {
+		L.DomUtil.remove(L.DomUtil.get('context-toolbar'));
```

Comment:
Can we just remember the instance somewhere and not try to query for it everywhere?